%YAML:1.0
---

# camera_CI_MAT: !!opencv-matrix
#   rows: 3
#   cols: 3
#   dt: d
#   data: [802.081304, 0.000000, 332.483229,0.000000, 795.871309, 148.056867,0.000000, 0.000000, 1.000000]

# camera_D_MAT: !!opencv-matrix
#   rows: 1
#   cols: 5
#   dt: d
#   data: [-0.068028, -0.034964, 0.002390, -0.001252, 0.000000]

# 图传相机参数，从老代码抄来的，可以重新标
camera_CI_MAT: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data: [9.89348020e+02, 0.00000000e+00, 1.29567949e+03,
          0.00000000e+00, 9.92649805e+02, 8.12444036e+02,
          0.00000000e+00, 0.00000000e+00, 1.00000000e+00]
camera_D_MAT : !!opencv-matrix
  rows: 1
  cols: 5
  dt: d
  data: [-0.20053016,  0.18680168,  0.00040175,  0.00299054, -0.18312982]




